src/autoComplete/AutoComplete.tsx

```diff
@@ -51,7 +51,8 @@ export interface AutoCompleteProps {
   readonly required?: boolean
   readonly error?: string | JSX.Element
   readonly selectedItemStatus?: ItemStatus
-  readonly shouldDisplayDivider?: boolean
+  readonly embeddedInSearchForm?: boolean
+  readonly loader?: JSX.Element
```

Comment:
Not sure we should expose the `loader` prop. We might end with random elements. Maybe just put a boolean like `shouldDisplayLoader`

Reply:
Well, I just saw the `embeddedInSearchForm` prop. You can just remove the loader prop as we'll always use the `Loader` component. Especially with your css targeting

Reply:
I agree with you.
Actually, I hesitated between keeping the loader outside of AutoComplete to have the lower impact or creating it in the AutoComplete and having only `embeddedInSearchForm`